src/test/java/uk/nhs/hee/trainee/details/api/ContactDetailsResourceTest.java

```diff
@@ -0,0 +1,90 @@
+package uk.nhs.hee.trainee.details.api;
```

Comment:
License

Reply:
Ugh, IntelliJ doesn't add it when creating test classes via class context menu it seems. I'll fix.